### PR TITLE
Small language fixes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: lua/terrortown/lang/en/infected.lua [hunk 1]
```diff
@@ -5,16 +5,16 @@ L[INFECTED.name] = "Infected"
...
+L["info_popup_" .. INFECTED.name] = [[Now it's your turn! Infect them all by killing them.]]
 L["body_found_" .. INFECTED.abbr] = "They were an Infected!"
 L["search_role_" .. INFECTED.abbr] = "This person was an Infected!"
 L["ev_win_" .. INFECTED.defaultTeam] = "The ill Infected won the round!"
 L["target_" .. INFECTED.name] = "Infected"
-L["ttt2_desc_" .. INFECTED.name] = [[The Infected needs to infect every player to win. He will infect other players by killing them.
-If a player gets infected, the player looks like a zombie and is also able to infect other players. So you can build up a whole army.
-But there is one thing you need to get in mind: If the host (the main infected player with a normal model) will die or disconnect, each player that the host infected will die.
+L["ttt2_desc_" .. INFECTED.name] = [[The Infected needs to infect every player to win. They infect other players by killing them.
+If a player gets infected, the player becomes a zombie and is able to infect other players. This allows zombies to build up a whole army.
```
Comment (Histalek) [inline lua/terrortown/lang/en/infected.lua]:
```suggestion
If a player gets infected, they become a zombie and are able to infect other players. This allows the infected to build up a whole army.
```

File: lua/terrortown/lang/en/infected.lua [hunk 2]
```diff
@@ -5,16 +5,16 @@ L[INFECTED.name] = "Infected"
...
+However, there is one thing you need to get in mind: If the host (the main infected player with a normal model) dies or disconnects, each player that the host directly infected will die.
```
Comment (Histalek) [inline lua/terrortown/lang/en/infected.lua]:
```suggestion
However, there is one thing you need to keep in mind: If the host (the original, non-zombie infected player) dies or disconnects, all zombies will die.
```
~~Also, don't ALL infected players die in that scenario? Do zombies who where infected by another zombie really not die?~~

**Review by Histalek — CHANGES_REQUESTED** [review]
Some small suggestions

File: lua/terrortown/lang/en/infected.lua (hunk 2 above)
Comment (TimGoll) [reply lua/terrortown/lang/en/infected.lua]: yes all zombies die. I'm 99% sure that "second grade" zombies die as well

Comment (Histalek) [reply lua/terrortown/lang/en/infected.lua]: thx, updated suggestion accordingly

File: lua/terrortown/lang/en/infected.lua (hunk 1 above)
Comment (ghost) [reply lua/terrortown/lang/en/infected.lua]: Excellent, I agree with these changes.

File: lua/terrortown/lang/en/infected.lua (hunk 2 above)
Comment (ghost) [reply lua/terrortown/lang/en/infected.lua]: Excellent, I agree with these changes.

**Review by Histalek — APPROVED** [review]

**Pull request merged**